CODEOWNERS

```
@@ -23,7 +23,7 @@
 
 * @open-telemetry/go-approvers
 
-CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu
+CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu @flc1125
```

```suggestion
CODEOWNERS @MrAlias @pellared @dashpole @XSAM @dmathieu
```

We're adding him as an approver, not a maintainer :)